package.json

```diff
@@ -7,7 +7,8 @@
     "build": "babel src  --out-dir production",
     "start": "node production/src/index.js",
     "dev": "nodemon --exec babel-node index.js",
-    "test": "nyc --reporter=html --reporter=text mocha  --require  @babel/register  src/tests/*.js --exit"
+    "test": "export NODE_ENV=test && nyc --reporter=html --reporter=text mocha  --require  @babel/register  src/tests/*.js --exit"
```

Comment:
Why this extra command `export NODE_ENV=test`

Reply:
I removed it, can remove it also from travis file?

Reply:
where I put global command?

Reply:
You can remove it here

Reply:
Yes done